types/stripe-js/radar.d.ts

```diff
@@ -0,0 +1,24 @@
+import * as api from '../api';
+
+/* A Radar Session is a snapshot of the browser metadata and device details that helps Radar make more accurate predictions on your payments. 
+This metadata includes information like IP address, browser, screen or device information, and other device characteristics. 
+You can find more details about how Radar uses this data by reading about how Radar performs [advanced fraud detection](https://stripe.com/docs/disputes/prevention/advanced-fraud-detection).
+*/
+export type RadarSessionPayload =
+  | {radarSession: Record<any, any>; error?: undefined}
+  | {radarSession?: undefined; error: LocalizedError};
```

Comment:
Instead of a new type def `LocalizedError`, can we use `StripeError` instead?

Reply:
Let me make sure it will conform to that type but definitely better that way if it does